src/network/network_content.h

```diff
@@ -82,6 +84,7 @@ class ClientNetworkContentSocketHandler : public NetworkContentSocketHandler, Co
 	bool Receive_SERVER_CONTENT(Packet *p) override;
 
 	ContentInfo *GetContent(ContentID cid);
+	const ContentInfo *GetContent(ContentID cid) const { return const_cast<ClientNetworkContentSocketHandler *>(this)->GetContent(cid); }
```

Comment:
Unless I'm missing something obvious, the existing GetContent method doesn't make any changes to the object and can be declared const without making this hacky extra function

Reply:
That's fixed now.